pkg/host/internal/sriov/sriov.go

```diff
@@ -93,7 +93,7 @@ func (s *sriov) ResetSriovDevice(ifaceStatus sriovnetworkv1.InterfaceExt) error
 	return nil
 }
 
-func (s *sriov) GetVfInfo(pciAddr string, devices []*ghw.PCIDevice) sriovnetworkv1.VirtualFunction {
+func (s *sriov) GetVfInfo(pciAddr string, ifaceType string, devices []*ghw.PCIDevice) sriovnetworkv1.VirtualFunction {
```

Comment:
i think you will not need ifaceType as we will need to get guid of rdma device regardless of the link.

Reply:
The outcome of this is that MAC address is populated with the GID for IB. Is that acceptable?

```
    - Vfs:
      - deviceID: 101c
        driver: mlx5_core
        guid: "01:00:00:00:00:00:00:00"
        mac: 00:00:00:e8:fe:80:00:00:00:00:00:00:00:00:00:00:00:00:00:02
        mtu: 4092
        name: ibp75s0f0v0
        pciAddress: 0000:4b:00.2
        vendor: 15b3
        vfID: 0
      - deviceID: 101c
        driver: mlx5_core
        guid: "05:00:00:00:00:00:00:00"
        mac: 00:00:01:e8:fe:80:00:00:00:00:00:00:00:00:00:00:00:00:00:00
        mtu: 4092
        name: ibp75s0f0v1
        pciAddress: 0000:4b:00.3
        vendor: 15b3
        vfID: 1
      deviceID: 101b
      driver: mlx5_core
      linkStatus: down
      linkType: IB
      mac: 00:00:05:91:fe:80:00:00:00:00:00:00:08:c0:eb:03:00:c1:33:66
      mtu: 4092
      name: ibp75s0f0
      numVfs: 2
      pciAddress: 0000:4b:00.0
      totalvfs: 2
      vendor: 15b3
```